### ISSUE TITLE
Wrong example in lab2

### ISSUE THREAD

**Issue by nlfox** [agamemnonc/iaml#3]
>*Tip*: Be careful, in python, unless you explictly say not to, assigment is typically just reference e.g.
```python
i = "I am i"
j = i
i = "It is I, i! j merely references me!"
print(j)
```
outputs:
```
"It is I, i! j merely references me!"
```

It's wrong, in Python, `str` is *immutable*,  see https://docs.python.org/2/faq/design.html?#why-are-python-strings-immutable. Immutable objects don't pass reference.  So the example should be shown in mutable objects like `list`

```python
i = ["I am i"]
j = i
i += ["It is I, i! j merely references me!"]
print(j)
```

**Comment by agamemnonc** [comment]
Yes, you are right. 

Thanks for pointing it out to us.

**Comment by agamemnonc** [comment]
Fixed 390a784f989829e672acb2bee229aa28c2e03cb8
